### DISCUSSION TITLE
Deprecation messages

### DISCUSSION THREAD

**Discussion by styyokuda** [idaholab/moose#24941] [Q&A General] [answered]
All,
I have received two messages for "deprecation" (please see below).  I don't understand them. Could someone provide me with examples to help me.

(1) *** Warning, This code is deprecated and will be removed in future versions:
“registerAll” in Modules is deprecated. Please update your *App.C file to call the new templated “registerAllObjects” method (e.g. ModulesApp::registerAllobject<MyApp>(…))

(2) For cylindrical coordinates, I use  
[./Problem]
  coord_type = 'RZ'
[../]
I received the message:
The parameter ‘coord_type’ is deprecated. Please use ‘Mesh/coord_type’ instead.

Please help.
S. Thomas

**Comment by GiudGiud** [comment]
Hello

the messages should be saying what to do. 
the first one is telling you to go in your src/base/….App.C and fix that one line

the second one is telling you to move that one line with coord_type from the problem block to the Mesh block

Guillaume

**Reply by styyokuda** [reply]
Hi Guillaume,
Attached is the error message on changing to `ModulesApp::registerAllobject(…).`
Please help. 
S. Thomas

![registerAllobject](https://github.com/idaholab/moose/assets/77596591/b5491dd7-6137-49e1-9bc6-d436ee8bb523)

**Reply by GiudGiud** [reply]
should be registerAllObjects with an s

should look something like:
```
  ModulesApp::registerAllObjects<ligerApp>(f, af, s);
```

**Reply by styyokuda** [reply, marked as answer]
Thank you for your patience with me.
S. Thomas
